### PR TITLE
 ✨ Source Freshcaller : Migrate Python CDK to Low-code CDK

### LINKED ISSUE #30893: :sparkles: Source Freshcaller : Migrate Python CDK to Low-code CDK
https://docs.airbyte.com/integrations/sources/freshcaller

**Rules for the Connectors Hackathon**
You can only work on one connector at a time
Make sure to resolve all suggestions from the Airbyte team
Must include all streams of the connector you’re migrating. Otherwise, it will not be accepted.
If you don’t commit anything within five days, the issue may be assigned to someone else.

### PR DESCRIPTION
**Pull request by aazam-gh**
## What
Fix #30893
Migrate Source Freshcaller  from Python CDK to Low-code CDK

## How
Using Low Code/No Code Connector Development Kit

## Recommended reading order
1. `manifest.yaml`
2. `schemas`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:generateScaffolds` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Comment by archangelic** [comment]
@aazam-gh it looks like this branch also has changes for source-zendesk-sell. Are you able to clean this PR so that the changes are limited to source-freshcaller?

**Comment by aazam-gh** [comment]
@archangelic sure I'll take a look at it and remove the Zendesk changes

File: airbyte-integrations/connectors/source-freshcaller/build.gradle [hunk 1]
```diff
@@ -0,0 +1,8 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+}
+
+airbytePython {
+    moduleDirectory 'source_freshcaller'
+}
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-freshcaller/build.gradle:8]: Remove this file.

File: airbyte-integrations/connectors/source-freshcaller/setup.py [hunk 2]
```diff
@@ -6,23 +6,24 @@
...
+    "airbyte-cdk~=0.1",
 ]
 
 TEST_REQUIREMENTS = [
-    "pytest~=6.1",
-    "pytest-mock~=3.6.1",
     "requests-mock~=1.9.3",
+    "pytest~=6.2",
+    "pytest-mock~=3.6.1",
+    "connector-acceptance-test",
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-freshcaller/setup.py:16]: Remove this import, not used anymore.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Missing update doc changelog. Remove changes to Zendesk Sell.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**